todo.js

```diff
@@ -15,3 +15,39 @@ Keep the responsibilities separated as best you can:
 4. Displaying information to the user
 5. Rather user input and taking the appropriate actions
 */
+const fs = require('fs');
+let todoList = fs.readFileSync("todos.txt","utf-8");
+let command = process.argv[2];
+let readLineSync = require("readLine-sync");
+let TODO = {};
+function list() {
+
+  if (command === "list")
+  {
+    console.log(todoList);
+  }
+}
+list();
+function append() {
+  if (command === "append")
+  {
+    let newTask = readLineSync.question("What would you like to add?");
+    console.log(todoList + newTask);
+  }
+}
+append();
+function delete() {
+  if (command === "delete")
+  {
+    let deleteTask = readLineSync.question("What would you like to delete?");
```

Comment:
See the README for how this feature is supposed to work. Let me know if you have questions about command line arguments.

Are you expecting a user to enter a number here or the text of the task? Read the README for a description of how this feature is meant to work. Typing the exact task description would be incredibly awkward on the command line (and very error-prone).

If you're entering a task number I'd expect you to convert the user input to a number.

See https://github.com/PMcS3/javascript-milestones/blob/conversions/conversions.js#L10 for an example of how to convert a string to a number.

If `todoList` is one big string and `deleteTask` is a number then you want to find the relevant line (as determined by counting `\n` characters in the string) and then delete everything between the relevant `\n` character and the next `\n` character.

Again, it'll be 100x easier if you represent the todo list as an array in memory.